pkg/endpoint/bpf.go

```diff
@@ -156,7 +156,7 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	}
 	fw.WriteString(" */\n\n")
 
-	if !e.PolicyCalculated && owner.PolicyEnabled() {
+	if !e.PolicyCalculated && e.Opts.IsEnabled(OptionPolicy) {
```

Comment:
This relaxes the policy constraints a bit. `UpdatePolicyEnforcement()` requires the labels to be resolved on the endpoint. This is not the case when the container starts right away as we can only request the labels after the container has been started. Therefore, this would allow a container to talk until the labels has been resolved. We need to continue checking the daemon config here and can only skip setting `DROP_ALL` if `daemon.config.EnablePolicy == Never` is set.